src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,51 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  position: relative;
+  max-width: 708px;
+  margin: 40px auto;
+  padding: 10px;
+
+  &__cell {
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 30px;
+    text-align: center;
+
+    display: block;
+    width: $block-size;
+    height: $block-size;
+    line-height: $block-size;
```

Comment:
Using line-height for alignment can lead to issues and is generally not considered the best practice.